### DISCUSSION TITLE
add newly server pool to initial minio cluster questions

### DISCUSSION THREAD

**Discussion by lddlww** [minio/minio#14742] [Q&A]
reference to
https://docs.min.io/minio/baremetal/installation/expand-minio-distribute

>MinIO requires each pool satisfy the deployment [erasure code](https://docs.min.io/minio/baremetal/concepts/erasure-coding.html#minio-erasure-coding) settings. Specifically the new pool topology must support a minimum of 2 x EC:N drives per [erasure set](https://docs.min.io/minio/baremetal/concepts/erasure-coding.html#minio-ec-erasure-set), where EC:N is the [Standard](https://docs.min.io/minio/baremetal/concepts/erasure-coding.html#minio-ec-storage-class) parity storage class of the deployment. This requirement ensures the new server pool can satisfy the expected SLA of the deployment.

i am confuse about why newly server pool set expected SLA that need to satisfy 2 x EC:N drivers per erasure set ？

**Comment by harshavardhana** [comment]
> reference to https://docs.min.io/minio/baremetal/installation/expand-minio-distribute
> 
> > MinIO requires each pool satisfy the deployment [erasure code](https://docs.min.io/minio/baremetal/concepts/erasure-coding.html#minio-erasure-coding) settings. Specifically the new pool topology must support a minimum of 2 x EC:N drives per [erasure set](https://docs.min.io/minio/baremetal/concepts/erasure-coding.html#minio-ec-erasure-set), where EC:N is the [Standard](https://docs.min.io/minio/baremetal/concepts/erasure-coding.html#minio-ec-storage-class) parity storage class of the deployment. This requirement ensures the new server pool can satisfy the expected SLA of the deployment.
> 
> i am confuse about why newly server pool set expected SLA that need to satisfy 2 x EC:N drivers per erasure set ？

Why should it not?

**Reply by lddlww** [reply]
if i set 1 x EC:N drivers per erasure set for newly pool, is it feasible？

**Reply by harshavardhana** [reply]
> if i set 1 x EC:N drivers per erasure set for newly pool, is it feasible？

No that is not supported
